### ISSUE TITLE
nit: lnav is not 100k lines of C++...

### ISSUE THREAD

**Issue by tstack** [rcoh/angle-grinder#33]
Howdy!  Angle-grinder is a great looking tool and I was excited to see it when I first stumbled onto it awhile ago!  I just noticed today that you mention lnav in the README, which is very nice of you.  But, I wanted to at least correct the line count as mentioned.  There is some generated code and header-only libraries that are checked in to lnav's repo to make it a bit easier to compile.  So, the quoted number of 100k lines is not really accurate, it's closer to 50k.  (That 50k count is a surprise to me, actually.  I haven't checked the line count in a while, thought it was closer to 20k...)

Respectfully, though, I'd ask you to remove the comparison altogether.  All that extra code does provide a lot of additional value.  It's not just a more complex version of angle-grinder.

**Comment by rcoh** [comment]
Hello! I'll fix the line count for sure. If you have an accurate comparison blurb I'd be happy to add it (or simply remove it from my README if you'd). Admittedly, I only added at when someone mentioned it to me as a similar tool and didn't dig in too deep. My apologies!

**Comment by rcoh** [comment]
Updated to:
```lnav is a full featured log file navigator in your terminal (with many more features than angle-grinder). It includes support for common log file formats out-of-the-box, generalized SQL queries on your logs, and auto-coloring.```

**Comment by tstack** [comment]
Thanks for the update, I've added a link to angle-grinder from lnav's README.md as well.

I don't think any comparison is really necessary.  I'm not here to compete with anybody.  I have a keen interest in log processing and I'm excited to see other approaches and ideas (so I can steal them... ;).  If anything, I could see directly integrating angle-grinder's guts with lnav.  One area where lnav is lacking is on-the-fly analysis.  I used sqlite for working with the data in lnav since I didn't want people to have to learn a new query language.  But, sqlite doesn't work for online processing.  The sumo language is pretty sane and has plenty of traction, so I'd be happy to have that functionality accessible through lnav.  Anyways...

I have to say, I'm jealous that you got to start out with Rust.  I keep wishing I had the free time to rewrite lnav in Rust.  I might try to satisfy my thirst for writing some rust code by working with angle-grinder.
